docs/wiki/The CI environment - Pipeline usage.md

```diff
@@ -0,0 +1,141 @@
+This section provides a guideline on how to use the CARML CI environment pipelines.
+
+---
+
+### _Navigation_
+
+
+- [Operate the module pipelines](#operate-the-module-pipelines)
+  - [Add a new module pipeline](#add-a-new-module-pipeline)
+- [Operate the dependency pipelines](#operate-the-dependency-pipeline)
+  - [Add a new dependency](#add-a-new-dependency)
+- [DevOps-Tool-specific](#devops-tool-specific-guidance)
+  - [GitHub workflows](#github-workflows)
+  - [Azure DevOps pipelines](#azure-devops-pipelines)
+
+---
+
+# Operate the module pipelines
+
+To validate any updates you did to a module template you can perform the following steps:
+
+1. (Optionally) Update the module's parameter files to reflect your changes.
+1. Push the local changes to the repository (using a branch that is not [main\|master]).
+1. On the DevOps platform, navigate to your pipelines and select the pipeline that was registered for the module you updated.
+1. Select the branch with your updated template.
+1. (Optionally) disable the `Remove deployed module` input parameter in case you don't want to apply the default behavior, and want to skip the deletion of the test-deployed resource to check it after deployment.
+1. (Optionally) adjust the `Publish prerelease module` flag in case you want to publish a prerelease version of your updated module from your development branch.
+1.  Trigger the pipeline.
+
+Once the pipeline concludes, it will either be in a green (success) or red (failed) state, depending on how the module performed.
+
+Pipeline logs are available for troubleshooting and provide detailed information in case of failures. If errors occur in the [Static validation](./The%20CI%20environment%20-%20Static%20validation) phase, you may only see the failed test and need to `expand` the error message. How this looks like depends on the [DevOps platform](#devops-tool-specific-guidance) you use.
+
+## Add a new module pipeline
+
+To add a new module pipeline we recommend to create a copy of a currently existing module pipeline and adjust all module-specific properties. The registration of the pipeline depends on the [DevOps platform](#devops-tool-specific-guidance) you're using.
```

Comment:
```suggestion
To add a new module pipeline we recommend to create a copy of a currently existing module pipeline and adjust all module-specific properties (e.g. triggers, module paths, etc.). The registration of the pipeline depends on the [DevOps platform](#devops-tool-specific-guidance) you're using.
```

Reply:
If you use e.g. you can avoid etc. 🦖

Reply:
```suggestion
To add a new module pipeline we recommend to create a copy of a currently existing module pipeline and adjust all module-specific properties, e.g. triggers and module paths. The registration of the pipeline depends on the [DevOps platform](#devops-tool-specific-guidance) you're using.
```

Reply:
Somehow I cannot commit your suggestion. Can you do that yourself?